### PR TITLE
Only show bookable slots that start on the next valid start date

### PR DESCRIPTION
**Pull request by AndrewVos**
(Which is two working days from the current day).

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@AndrewVos looks ok to me!

File: app/assets/javascripts/calendars/appointment-availability.es6 [hunk 1]
```diff
@@ -91,6 +92,10 @@
       this.selectedEventColour = 'green';
     }
 
+    goToNextValidDay() {
+      this.$el.fullCalendar('gotoDate', moment(this.$el.data('start-date')))
```
Comment (Guntrisoft) [inline app/assets/javascripts/calendars/appointment-availability.es6]: https://fullcalendar.io/docs/current_date/defaultDate/

Comment (AndrewVos) [inline app/assets/javascripts/calendars/appointment-availability.es6]: Yep! Didn't find that

File: spec/models/bookable_slot_spec.rb [hunk 2]
```diff
@@ -40,6 +40,22 @@ def result
         )
       end
 
+      it 'exludes bookables slots that start too soon' do
```
Comment (Guntrisoft) [inline spec/models/bookable_slot_spec.rb]: <trolling>missing a `c` from excludes</trolling>

File: app/assets/javascripts/calendars/appointment-availability.es6 (hunk 1 above)
Comment (Guntrisoft) [inline app/assets/javascripts/calendars/appointment-availability.es6]: looking good

**Comment by Guntrisoft** [comment]
@AndrewVos apart from typo, all good 👍

**Pull request merged**